drivers/mge-hid.c

```diff
@@ -909,21 +908,26 @@ static const char *eaton_input_eco_mode_auto_on_fun(double value)
 	}
 
 	/* Check if input.eco.switchable is normal and set it to 'ECO' */
-   if (strcmp(eco_switchable_str, "normal")) {
+    if (strcmp(eco_switchable_str, "normal")) {
 	    check_eco_result = eaton_input_eco_mode_check_range(value);
```

Comment:
@jimklimov please confirm here we should call function to check range becouse : 
when `setvar("input.eco.switchable", "ECO");` its only set variable but not calling function `eaton_input_eco_mode_check_range` like here, same for bypass?

https://github.com/networkupstools/nut/blob/858f615a32aaef3497442eae0155a773953123dd/drivers/mge-hid.c#L1709-L1710

https://github.com/networkupstools/nut/blob/858f615a32aaef3497442eae0155a773953123dd/drivers/mge-hid.c#L783-L789